x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.test.tsx

```diff
@@ -85,14 +85,34 @@ describe('AlertSummaryView', () => {
       expect(queryByTestId('summary-view-guide')).not.toBeInTheDocument();
     });
   });
+  test('Network event renders the correct summary rows', () => {
+    const renderProps = {
+      ...props,
+      data: mockAlertDetailsData.map((item) => {
+        if (item.category === 'event' && item.field === 'event.category') {
+          return {
+            ...item,
+            values: ['network'],
+            originalValue: ['network'],
+          };
+        }
+        return item;
+      }) as TimelineEventsDetailsItem[],
+    };
+    const { container } = render(
+      <TestProvidersComponent>
+        <AlertSummaryView {...renderProps} />
+      </TestProvidersComponent>
+    );
+    expect(container.querySelector('div[data-test-subj="summary-view"]')).toMatchSnapshot();
```

Comment:
Thanks for updating the tests. Would you be open to making this an inline snapshot or use a `queryByText` to check for specific items in the component? Snapshots in a separate file tend to get overwritten without much thought 😓 , so ideal to keep it in the same file where possible or explicitly test for the content in the component.

Reply:
I would not change it to inline snapshots since they're huge. But I really like the idea of using `queryByText` here 👍